main.js

```diff
@@ -9,6 +10,11 @@ for(var c = 0; c < phrase.length; c++) {
   array.push(obj)
 }
 
+var applicationState = {
+  charachters: 'grumpy wizards make toxic brew for the evil queen',
```

Comment:
This should be your array of objects. Put the character objects here instead of in another variable. We want one object to contain all of the information about the app.